zulint/command.py

```diff
@@ -44,13 +44,28 @@ def add_default_linter_arguments(parser: argparse.ArgumentParser) -> None:
     parser.add_argument('--fix',
                         action='store_true',
                         help='Automatically fix problems where supported')
+    parser.add_argument('--jobs', '-j',
+                        default=-1,
+                        type=int,
+                        help='Limit number of parallel jobs')
 
 def split_arg_into_list(arg: str) -> List[str]:
     return [linter for linter in arg.split(',')]
 
-def run_parallel(lint_functions: Mapping[str, Callable[[], int]]) -> bool:
-    pids = []
+def run_parallel(lint_functions: Mapping[str, Callable[[], int]], jobs: int) -> bool:
+    pids: Set[int] = set()
+    failed = False
+
     for name, func in lint_functions.items():
+        while jobs == 0:
+            pid, status = os.wait()
+            if pid in pids:
+                if status != 0:
+                    failed = True
+                pids.remove(pid)
+                jobs += 1
+
+        jobs -= 1
```

Comment:
May be worth a comment here about how the `jobs=-1` strategy works.  Otherwise lgtm.